### DISCUSSION TITLE
Gatsby /w Jest doesn't recognize alias

### DISCUSSION THREAD

**Discussion by uh-oh-404** [gatsbyjs/gatsby#28298] [Help]
#24336  Description

I try to run tests with files that use alias and I get errors due to it not being able to recognize the alias

### Steps to reproduce

Create unit test and run the against a component using aliases

### Expected result

Should run the test

### Actual result

It errored

### Environment

  System:
    OS: macOS 11.0.1
    CPU: (12) x64 Intel(R) Core(TM) i7-9750H CPU @ 2.60GHz
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 12.18.3 - /usr/local/bin/node
    npm: 6.14.6 - /usr/local/bin/npm
  Languages:
    Python: 2.7.16 - /usr/bin/python
  Browsers:
    Chrome: 87.0.4280.67
    Safari: 14.0.1
  npmPackages:
    gatsby: ^2.25.0 => 2.26.1 
    gatsby-image: ^2.4.21 => 2.5.0 
    gatsby-plugin-alias-imports: ^1.0.5 => 1.0.5 
    gatsby-plugin-apollo: ^3.0.2 => 3.0.2 
    gatsby-plugin-breadcrumb: ^10.0.0 => 10.0.0 
    gatsby-plugin-prettier-eslint: ^1.0.4 => 1.0.4 
    gatsby-plugin-sass: ^2.4.0 => 2.5.1 
    gatsby-plugin-sharp: ^2.7.0 => 2.8.0 
    gatsby-plugin-styled-components: ^3.3.14 => 3.4.0 
    gatsby-plugin-transition-link: ^1.20.5 => 1.20.5 
    gatsby-source-filesystem: ^2.4.0 => 2.5.0 
    gatsby-transformer-json: ^2.4.15 => 2.5.0 
    gatsby-transformer-sharp: ^2.5.21 => 2.6.0 
  npmGlobalPackages:
    gatsby-cli: 2.12.113

**Comment by LekoArts** [comment]
I'd suggest asking in Jest specific places, this isn't most likely something where Gatsby is the problem. A small reproduction of your problem would be great :)
